src/api/index.js

```diff
@@ -0,0 +1,88 @@
+import { Router, json } from 'express'
+import ecc from 'eosjs-ecc'
+import {Serialize, Numeric} from 'eosjs'
+import base64url from 'base64url'
+import cbor from 'cbor'
+import util from 'util';
+
+export default () => {
+  const private_key_wif = process.env.API_SERVER_PRIVATE_KEY
+  const api = Router()
+
+  const decodeWebauthnPublicKey = (webauthnPublicKey) => {
+    const attestationBuffer = base64url.toBuffer(webauthnPublicKey.attestationObject)
+    const attestation = cbor.decodeFirstSync(attestationBuffer)
+    const authdata = attestation.authData
+    const flags = authdata.readUInt8(32)
+    const credentialIDLength = authdata.readUInt16BE(53)
+    const credentialID = authdata.slice(55, 55 + credentialIDLength)
+    const COSEPublicKeyBuffer = authdata.slice( 55 + credentialIDLength )
+    const COSEPublicKey = cbor.decodeFirstSync(COSEPublicKeyBuffer)
+
+    console.log(COSEPublicKey);
+    const x   = COSEPublicKey.get(-2)
+    const y   = COSEPublicKey.get(-3)
+    console.log(Buffer.from(x).toString('hex'))
+    console.log(Buffer.from(y).toString('hex'))
+
+    const rpId = 'localhost'
+    const presence = ((flags)=>{
+      if (flags & 0x04)
+        return 2
+      else if (flags & 0x01)
+        return 1
+      else
+        return 0
+    })(flags)
+
+    const ser = new Serialize.SerialBuffer({textEncoder: new util.TextEncoder(), textDecoder: new util.TextDecoder()})
+    ser.push(2)
+    ser.push((y[31] & 1) ? 3 : 2)
+    ser.pushArray(x)
+    ser.push(presence)
+    ser.pushString(rpId)
+    const eosioPubkey = ser.asUint8Array()
+
+    console.log(Buffer.from(eosioPubkey).toString('hex'))
+    return {eosioPubkey, credentialID}
+  }
+
+  const users = {}
+
+  api.post( '/generateRentChallenge', json(), (req, resp) => {
+    const name = req.body.accountName
+    const propertyName = req.body.propertyName
```

Comment:
I know this is just an example, but are there regExs we could use to pseudo-sanitize these fields?  Maybe a max length and character set, eg. for property [A-Za-z0-9,. ]{100}?